packages/toolkit/src/test/srcShared/fs.test.ts

```diff
@@ -24,17 +24,17 @@ describe('FileSystem', function () {
 
     before(async function () {
         fakeContext = await FakeExtensionContext.create()
+        sandbox = Sinon.createSandbox()
         await deleteTestRoot() // incase a previous test run failed to clean
     })
 
     beforeEach(async function () {
         await makeTestRoot()
-        sandbox = Sinon.createSandbox()
+        sandbox.restore()
     })
 
     afterEach(async function () {
         await deleteTestRoot()
-        sandbox.restore()
     })
```

Comment:
Isn't this still needed? Could it affect later tests? https://sinonjs.org/releases/latest/sandbox/